sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -996,6 +996,30 @@ def test_negative_fractions():
     expr = Mul(Rational(2, 3), Rational(5, 7), evaluate=False)
     assert pretty(expr) == "2/3*5/7"
     assert upretty(expr) == "2/3⋅5/7"
+    expr = Mul(x + y, Rational(1, 2), evaluate=False)
+    assert pretty(expr) == "(x + y)*1/2"
+    assert upretty(expr) == "(x + y)⋅1/2"
+    expr = Mul(Rational(1, 2), x + y, evaluate=False)
+    assert pretty(expr) == "x + y\n-----\n  2  "
+    assert upretty(expr) == "x + y\n─────\n  2  "
+    expr = Mul(S.One, x + y, evaluate=False)
+    assert pretty(expr) == "1*(x + y)"
+    assert upretty(expr) == "1⋅(x + y)"
+    expr = Mul(x - y, S.One, evaluate=False)
+    assert pretty(expr) == "(x - y)*1"
+    assert upretty(expr) == "(x - y)⋅1"
+    expr = Mul(Rational(1, 2), x - y, S.One, x + y, evaluate=False)
+    assert pretty(expr) == "1/2*(x - y)*1*(x + y)"
+    assert upretty(expr) == "1/2⋅(x - y)⋅1⋅(x + y)"
+    expr = Mul(x + y, Rational(3, 4), S.One, y - z, evaluate=False)
+    assert pretty(expr) == "(x + y)*3/4*1*(y - z)"
+    assert upretty(expr) == "(x + y)⋅3/4⋅1⋅(y - z)"
+    expr = Mul(x + y, Rational(1, 1), Rational(3, 4), Rational(5, 6),evaluate=False)
+    assert pretty(expr) == "(x + y)*1*3/4*5/6"
```

Comment:
Shouldn't `Rational(1,1)`, strictly, print as `1/1`? What about `Rational(2,4)`? Or is that not kept with `evaluate=False`?

Reply:
@oscargus I'm not sure of `Rational(1, 1)` strictly print as `1/1` , since the equation here is printed similar to the output of master except that brackets were added in the beginning of the expression.

Reply:
`Rational(1, 1)` becomes `Integer(1)` which becomes `S.One`. There is no "unevaluated Rational" but you can do the same with a Pow if you want:
```python
In [72]: Pow(1, -1, evaluate=False)
Out[72]: 
1
─
1
```